django_tablib/base.py

```diff
@@ -54,7 +54,10 @@ def _getattrs(self, obj):
                 if hasattr(obj, 'get_{0}_display'.format(attr)):
                     value = getattr(obj, 'get_{0}_display'.format(attr))()
                 else:
-                    value = getattr(obj, attr)
+                    if '.' in attr:
+                        value = resolve_related_value(obj, attr)
```

Comment:
Since `"foo".split(".") == ["foo"]` it seems we could eliminate the conditional here, and pass `attr` to the helper without inspection.

Reply:
ahoy! good call. so easy to miss things like this when in a silo